lib/transactions.js

```diff
@@ -83,12 +88,55 @@ export const renderDetailsString = ({
     intl.locale,
   );
 
-  const detailString = [amountString, hostFeeString, platformFeeString, paymentProcessorFeeString];
-  if (taxAmount) {
-    detailString.push(formatFee(taxAmount, amount, taxInfo?.id || 'Taxes', taxInfo?.percentage, intl.locale));
+  if (kind === TransactionKind.EXPENSE) {
+    const expenseAmount = formatCurrency(Math.abs(_amount.valueInCents), _amount.currency, { locale: intl.locale });
+    const expensePaymentProcessorFee = formatCurrency(
+      Math.abs(paymentProcessorFee.valueInCents),
+      paymentProcessorFee.currency,
+      { locale: intl.locale },
+    );
+    const netExpenseAmount = formatCurrency(Math.abs(_netAmount.valueInCents), amount.currency, {
+      locale: intl.locale,
+    });
+    const payee = toAccount.name;
+    const payer = fromAccount.name;
+    return (
+      <React.Fragment>
+        <Container mb={1}>
+          {isCredit ? (
+            <FormattedMessage defaultMessage="Amount from {payer}: {expenseAmount}" values={{ payer, expenseAmount }} />
+          ) : (
+            <FormattedMessage defaultMessage="Expense Amount: {expenseAmount}" values={{ expenseAmount }} />
+          )}
```

Comment:
The labels here are likely false in some key cases and can surely be improved. 

Could you double check if fees paid by payee/collective? Also refunds?

How are they looking in all test cases?

Reply:
This PR has been updated. Feel free to let me know if you see any issues. Thanks.